docs/ci-integration/guides/google-cloud-build.md

```diff
@@ -72,7 +72,7 @@ It is also possible to perform these steps via the CLI; the steps are [also deta
 {% hint style='danger' %}
 ##### Note
 
-This example is not production ready, and is intended to showcase configuration needed to get Earthly off the ground. If you run into any issues, or need help, [don't hesitate to reach out](https://github.com/EarthBuild/earthbuild/issues/new)!
+This example is not production ready, and is intended to showcase configuration needed to get Earthly off the ground. If you run into any issues, or need help, [don't hesitate to reach out](https://github.com/earthly/earthly/issues/new)!
```

Comment:
this one shouldn't change either.

If you're doing a find + replace I think you need to check the part after the repo name too.

`/pull` and `/issue` should stay the same (`earthly/earthly`). `/issues/new` and blobs should move to `earthbuild/earthbuild`